src/main/java/pl/jakubkozlowski/learning/firststeps/controller/ChampionController.java

```diff
@@ -16,38 +17,36 @@
 @RequestMapping(path = BASE_PATH)
 public class ChampionController {
 
-    private ChampionMapper championMapper;
-
-    @Autowired
-    public ChampionController(ChampionMapper championMapper) {
-        this.championMapper = championMapper;
+    private ChampionService championService;
+    public ChampionController(ChampionService championService){
```

Comment:
Autowired on constructor missing. In new spring it is in many cases ommitable but do not get used to it.